met-api/tests/unit/models/test_widget_documents.py

```diff
@@ -55,6 +55,7 @@ def test_get_all_by_widget_id(session):
     session.commit()
 
     expected_docs = WidgetDocumentsModel.get_all_by_widget_id(widget.id)
+    all(expected_docs[i].id <= expected_docs[i + 1].id for i in range(len(expected_docs) - 1))
```

Comment:
what does this do?

Reply:
it checks if the expected docs are order in ascending order